clients/java/dkv-client/src/test/resources/dkv_config.yaml

```diff
@@ -23,3 +22,11 @@ nexus-repl-timeout : 5
 nexus-lease-based-reads : true
 nexus-snapshot-count : 100
 nexus-snapshot-catchup-entries : 5
+discovery-service:
+  ServerConfig:
```

Comment:
could we use the same naming format?  (dashed format)

Reply:
made the changes